android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1940,6 +2058,34 @@ private List<String> getStringArrayList(@Nullable ReadableArray readableArray)
         return list;
     }
 
+    private WritableArray getWritableArray(@Nullable List<String> list)
+    {
+        if ( list == null ) return null;
+
+        WritableArray array = Arguments.createArray();
+
+        for ( String item : list )
+        {
+            array.pushString( item );
+        }
+
+        return array;
+    }
+
+    private WritableArray convertReadableToWritableArray(@Nullable ReadableArray readableArray)
```

Comment:
Same comment as above.

Reply:
Can't we just use one list type? This is making the code harder to maintain by doing so many different list type conversions `getWritableArray()` and `convertReadableToWritableArray()`. I am pretty sure we can write our code, and eliminate the need for both.

Reply:
Removed `getWritableArray()` and `convertReadableToWritableArray()`.